examples/plotting/file/pixel_fun.py

```diff
@@ -0,0 +1,37 @@
+#Just for fun, taking a jpg image and plotting it pointillistic style on a bokeh canvas using circle glyphs!
+#Requires PIL!
+import Image
```

Comment:
This is extreme WIP, the real direction for this is to have a hybrid dev example that pulls video from HTML media APIs and updates a pixelated image as fast as is feasible. I would expect the PIL dependency to disappear entirely. This could also be a testbed for adding a low-level "function interface" that just passes JS along so that it could actually be rendered as a static document with no false server like the spectrogram. This simple kind of interface is what mpld3 does to expose plugins and I think we need to have *something* along these lines, at least to start with. (@pzwang please chime in with your thoughts on this too)